pkg/engine/imageVerify.go

```diff
@@ -219,27 +220,27 @@ func (iv *imageVerifier) verifySignature(imageVerify *v1.ImageVerification, imag
 	return ruleResp, digest
 }
 
-func (iv *imageVerifier) patchDigest(imageInfo context.ImageInfo, digest string, ruleResp *response.RuleResponse) {
+func (iv *imageVerifier) patchDigest(path string, imageInfo imageutils.ImageInfo, digest string, ruleResp *response.RuleResponse) {
 	if imageInfo.Digest == "" {
-		patch, err := makeAddDigestPatch(imageInfo, digest)
+		patch, err := makeAddDigestPatch(path, imageInfo, digest)
 		if err != nil {
-			iv.logger.Error(err, "failed to patch image with digest", "image", imageInfo.String(), "jsonPath", imageInfo.JSONPointer)
+			iv.logger.Error(err, "failed to patch image with digest", "image", imageInfo.String(), "jsonPath", path)
 		} else {
 			iv.logger.V(4).Info("patching verified image with digest", "patch", string(patch))
 			ruleResp.Patches = [][]byte{patch}
 		}
 	}
 }
 
-func makeAddDigestPatch(imageInfo context.ImageInfo, digest string) ([]byte, error) {
+func makeAddDigestPatch(path string, imageInfo imageutils.ImageInfo, digest string) ([]byte, error) {
 	var patch = make(map[string]interface{})
 	patch["op"] = "replace"
-	patch["path"] = imageInfo.JSONPointer
+	patch["path"] = path
```

Comment:
@samj1912 this change is breaking the `patchDigest` functionality, I tried to revert back to `JSONPointer` but I'm not getting the expected output as before and it looks like I'm doing something wrong, can you please look into this and guide me to make this work. Thanks 🤗

Reply:
It is fixed in https://github.com/kyverno/kyverno/pull/3596/files

Reply:
Thanks @samj912 for informing. I wasted a lot of time to figure it out